### ISSUE TITLE
Emoji box off truncated with the dash in a word.

### ISSUE THREAD

**Issue by vhashimotoo** [desktop/desktop#6806]
## Description

Emoji box off truncated with the dash in a word.

## Version

* GitHub Desktop: 1.6.1

* Operating system: Microsoft Windows [Version 10.0.17134.556]

## Steps to Reproduce

1. Create a commit message of more than 72 symbols.
2. Add a dash in a word. (Example: da-sh)
3. Try to add emoji by `:`.

### Expected Behavior
Emoji box not truncated.

### Actual Behavior

Emoji box truncates.

### Screencast of repro

![1](https://user-images.githubusercontent.com/24681191/52516411-43905f00-2c33-11e9-898b-a815239c5f80.gif)

**Comment by steveward** [comment]
Thanks for the detailed report @HashimotoYT! I was able to reproduce this bug as well. I'm going to mark it as a `priority-3` for the team to track.

**Comment by davidlloyduk** [comment]
I had a look at this and logged (https://github.com/component/textarea-caret-position/issues/63) - its returning an invalid coordinates value in 'autocompleting-text-input.tsx' (line 153) after calling 'getCaretCoordinates'. Once a dash '-' is added with another character the caret is calculated incorrectly , so coordinates.left is invalid (it jumps from like 300 to 20). Will await response on logged issue on textarea-caret-position before fixing anything. I debugged it on their repo and its some very strange span offsetLeft/offsetTop values being returned for an internal span. I tried equals and other characters and they all work fine. Something very odd going on there!
